### ISSUE TITLE
Installation to drive other than C: may not be working

### ISSUE THREAD

**Issue by GregBakker** [disguise-one/hap-encoder-adobe-cc#27]
Issue report:

```
I´m trying the CC suite 2019. I´ve simply
run the installer for the exporter and the
HAP-codec and was expecting to see HAP
in Adobe Media encoder but can´t.
I´m on win10 Home.

I´ve installed the exporter and the hap
codec on an additional internal drive
(not C) could this be why it doesn´t
work? The installer gave the option to
install elsewhere so should be fine.
```

More information is being requested [has the default installation been tried?]

**Comment by afelicirai** [comment]
The installation path refers only to the destination of doc files and the uninstaller.exe
The plugin (.epr) goes to common "Mediacore" folder to function properly.
The presets goes to user Adobe presets folder ( e.g.  ~\Documents\Adobe\Adobe Media Encoder\13.0\Presets ), but I have suggested to install the presets in the system folder (both Media Encoder and Premiere), in the correct folder ( for example c:\Program Files\Adobe\Adobe Media Encoder CC 2019\MediaIO\systempresets\48415059_686170 )

**Comment by GregBakker** [comment]
You're absolutely correct - the nominated location shouldn't really affect the install.

Adobe used to nominate where to find and put things with registry keys, but they don't work as described in the docs - and after a lot of searching I couldn't find better spots for things related to registry keys that *are* set. Feedback from Adobe has been to put it in the current location, with no mention of registry keys. We have similar problems on Mac vs Windows.

We should be wary of placing things in the system folder; even the correct one - the CC applications could ignore / upgrade / relocate what's being looked for there in successive releases - just a guess though really.

Maybe CC itself can be located elsewhere - I don't recall if that's possible, or if it would cause this plugin problems.

**Comment by GregBakker** [comment]
We're about to be testing 1.2.0; this is a cannot reproduce atm
